packages/perseus/src/styles/styles.less

```diff
@@ -420,24 +417,17 @@
 
     .math(@blockMathTextSize; @blockMathLineHeight;
           @inlineMathTextSize; @inlineMathLineHeight) {
-        :is(.katex, mjx-container):not(.mafs-graph *) {
+        mjx-container:not(.mafs-graph *) {
             font-size: @inlineMathTextSize;
             line-height: @inlineMathLineHeight;
             color: @gray17;
         }
 
-        .perseus-block-math .katex,
         .perseus-block-math mjx-container {
             font-size: @blockMathTextSize;
             line-height: @blockMathLineHeight;
         }
 
-        // HACK(charlie): I am ashamed. This seems to be the only way to revert
```

Comment:
Nice 👍

Reply:
I don't even know which Charlie wrote this, but whoever they are, I hereby absolve them of their sins.